.gitignore

```diff
@@ -1,4 +1,5 @@
 /target
+/micro-rdk-installer/target
```

Comment:
wit the workspace update there shouldn't be a target folder created under micro-rdk-installer. Under which condition do you end up in this situation?

Reply:
ack. i'll double check. this was an old fork